learn2track/factories.py

```diff
@@ -165,11 +165,12 @@ def loss_factory(hyperparams, model, dataset, loss_type=None):
 
     elif hyperparams['model'] == 'ffnn_regression':
         if loss_type == 'expected_value':
-            from learn2track.models.ffnn_regression import UndirectedL2Distance
-            return UndirectedL2Distance(model, dataset, hyperparams['normalize'])
-        elif loss_type == 'l2_distance':
-            from learn2track.models.ffnn_regression import L2Distance
-            return L2Distance(model, dataset, hyperparams['normalize'])
+            if hyperparams['predict_offset']:
+                from learn2track.models.ffnn_regression import L2Distance
+                return L2Distance(model, dataset, hyperparams['normalize'])
```

Comment:
Didn't we come to the conclusion that we should use UndirectedL2Distance? (There was a 0.01 difference in favor of it when you checked)